runtime/development/src/lib.rs

```diff
@@ -1873,6 +1856,26 @@ impl pallet_order_book::Config for Runtime {
 	type Weights = weights::pallet_order_book::WeightInfo<Runtime>;
 }
 
+pub struct SourceConverter;
+impl
+	sp_runtime::traits::Convert<
+		(Vec<u8>, Vec<u8>),
+		Result<cfg_types::domain_address::DomainAddress, DispatchError>,
+	> for SourceConverter
+{
+	fn convert(
+		_source: (Vec<u8>, Vec<u8>),
+	) -> Result<cfg_types::domain_address::DomainAddress, DispatchError> {
+		Err(DispatchError::Other("No supported domains yet!"))
```

Comment:
This is what you mentioned that you would change still right?

Reply:
Changed in latest version